docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -170,17 +170,25 @@ Now, select **Yes** to enable SPI:
 Finally, select **Finish**.
 Restart your Pi using `sudo reboot` to make these changes take effect.
 
-### Test your soil moisture readings on your Pi
+### Install the Python SDK
 
-Next, install the Adafruit ADC library [`Adafruit_CircuitPython_MCP3xxx`](https://github.com/adafruit/Adafruit_CircuitPython_MCP3xxx/) on your Pi.
+Next, set up a virtual environment and install the SDK.
+(If you want to read more on virtual environments, check out [the documentation](https://docs.viam.com/build/program/python-venv/).)
```

Comment:
However this is also out of context and would be better placed alongside the commands you're runnign to create the virtual environment (lines 185+)